### PR TITLE
Add Mac OS X support to Chromium testing

### PR DESCRIPTION
**Pull request by TheNavigat**

### CODE REVIEW

**Comment by TheNavigat** [comment]
You're correct. I've fixed the issues.

Additionally, sorry for mistakenly having your comments deleted. I updated the same commit, not sure why did that change the hash.

A screenshot of your comments is attached.

Sorry again.

selection_026

File: test/chrome/script.py [hunk 1]
```diff
@@ -18,6 +18,14 @@
 from selenium.webdriver.common.by import By
 from selenium.common.exceptions import WebDriverException
 
+if system.platform.startswith("linux"):
```
Comment (semenko) [inline test/chrome/script.py]: Probably sys.platform, not system (undefined, I bet)

Comment (TheNavigat) [inline test/chrome/script.py]: Oh God... Sorry... My bad. That's why I need a mentor, I guess.

**Comment by semenko** [comment]
As a general Python tip: You should try running pylint (or pep8) on your code -- which flags some useful syntax tips & other errors.

e.g. Here's what pylint complains about (with my custom .pylintrc file), limited to non-stylistic stuff:

```
C: 45, 0: No space allowed around keyword argument assignment
	driver = webdriver.Chrome(chromedriver_path, chrome_options = chromeOps)
                                                             ^ (bad-whitespace)
C: 63, 0: Unnecessary parens after 'elif' keyword (superfluous-parens)
C:  1, 0: Missing module docstring (missing-docstring)
E: 21, 3: Undefined variable 'system' (undefined-variable)
E: 23, 5: Undefined variable 'system' (undefined-variable)
C: 29, 0: Missing class docstring (missing-docstring)
W: 29, 0: Class has no __init__ method (no-init)
C: 29, 0: Old-style class defined. (old-style-class)
R: 29, 0: Too few public methods (0/2) (too-few-public-methods)
W: 55, 2: Unreachable code (unreachable)
W: 13, 0: Unused import glob (unused-import)
W: 13, 0: Unused import time (unused-import)
W: 17, 0: Unused import expected_conditions (unused-import)
W: 15, 0: Unused import Options (unused-import)
W: 13, 0: Unused import traceback (unused-import)
W: 13, 0: Unused import os (unused-import)
W: 18, 0: Unused import By (unused-import)
W: 16, 0: Unused import WebDriverWait (unused-import)
```

**Comment by semenko** [comment]
Cool -- generally LGTM -- unless you feel like making any of those pylint / pep8 cleanups.

(I'll merge this later tonight when I can get to a Mac for testing.)

**Comment by TheNavigat** [comment]
Yep, I'll clean it up a bit more, but it should be done in 30 minutes or so.

One small question, what's "LGTM"? :D

**Comment by semenko** [comment]
See: https://sites.google.com/a/chromium.org/dev/glossary

**Comment by TheNavigat** [comment]
Oh, okay. Thanks!

**Comment by semenko** [comment]
Hey, cool, works great! Happy to merge if you fix the conflict. :)

**Comment by TheNavigat** [comment]
I'm not sure how to do that.

screenshot_2015-08-31-11-56-05 2

That appears on this page but the conflicts themselves are not there.

**Comment by reedy** [comment]
Github has instructions for it, you've gotta do it locally

```
Checkout via command line

If you cannot merge a pull request automatically here, you have the option of checking it out via command line to resolve conflicts and perform a manual merge.
 HTTPS
 Git
Patch

https://github.com/TheNavigat/https-everywhere.git

Step 1: From your project repository, check out a new branch and test the changes.

git checkout -b TheNavigat-issue-2721 master
git pull https://github.com/TheNavigat/https-everywhere.git issue-2721
Step 2: Merge the changes and update on GitHub.

git checkout master
git merge --no-ff TheNavigat-issue-2721
git push origin master
```

**Comment by semenko** [comment]
Awesome -- thanks!

**Pull request merged**